### PR TITLE
Pds registry app 263 docs docker compose tl

### PR DESCRIPTION
**Pull request by tloubrieu-jpl**
## 🗒️ Summary
make the documentation a bit more user oriented by shifting the focus away from docker-compose when it is not useful to the user.

If you are good with that and if you don't catch typo and errors we can merge in your branch and then your branc in main.

### CODE REVIEW

File: docs/source/about.rst [hunk 1]
```diff
@@ -2,34 +2,61 @@
...
 
-The high level architecture of PDS Registry and its main components is shown below.
+PDS Registry Service enables tracking, searching, auditing, locating, and maintaining artifacts within the system.
+These artifacts can range from data files and label files, schemas, dictionary definitions for objects and elements, services, etc...
+
+The PDS Registry Service is enabled by a complex architecture of components deployed on the AWS Cloud by Engineering Node and on premises by the PDS Discipline Nodes.
+
+The high level architecture of the PDS Registry Service is shown below:
+
+.. image:: _static/images/registry_service.png
```
Comment (ramesh-maddegoda) [inline docs/source/about.rst:15]: This image is not visible

File: docs/source/admin/tasks.rst [hunk 2]
```diff
@@ -2,6 +2,10 @@
 Admin Tasks
 ===========
 
+Admin tasks describes processes for the intialization/deletion of the registry and reference dictionnary updates.
```
Comment (ramesh-maddegoda) [inline docs/source/admin/tasks.rst:5]: Minor typo: "Admin **tasks** **describes**" should be "Admin **tasks** **describe**"

File: docs/source/cloud/architecture.rst [hunk 3]
```diff
@@ -0,0 +1,66 @@
+===========================================
+The Registry API Service Cloud Architecture
+===========================================
+
+On-Prem vs. Cloud
+*****************
+
+In a typical on-premise deployment, the PDS Registry Application is composed, server-side of
```
Comment (ramesh-maddegoda) [inline docs/source/cloud/architecture.rst:8]: I feel it is better to reword this (from line 8 to 11).

File: docs/source/cloud/architecture.rst [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+of containerized applications (tasks) without requiring explicit
+provisioning of compute resources. The images for these application
+containers are placed in and pulled from the Elastic Container Registry
+(ECR). Each service instance connects to Elasticsearch, with the
+necessary credentials injected into each container by the Secrets
+Manager. Other non-sensitive run-time configuration is provided by the
+Systems Manager Parameter Store.
+
+Amazon's content delivery network, Cloudfront, is placed in front of
+the Elastic Load Balancer (ELB). Cloudfront provides high-speed, low-latency
```
Comment (ramesh-maddegoda) [inline docs/source/cloud/architecture.rst:42]: I feel it is better to reword "Cloudfront provides high-speed, low-latency access while also ensuring unintention or malicious bulk downloading of data is avoided". 

Suggestion (may be not the best suggestion) : "Cloudfront provides high-speed, low-latency access, while also ensuring the avoidance of  **unintentional** or malicious bulk downloading of data".

File: docs/source/index.rst [hunk 5]
```diff
@@ -5,14 +5,12 @@
...
-The core functionality for this component is satisfied by `OpenSearch <https://opensearch.org/>`_.
+PDS Registry provides functionality for tracking, searching, auditing, locating, and maintaining artifacts within the system.
+These artifacts can range from data files and label files, schemas, dictionary definitions for objects and elements, services, etc...
 
-The high level architecture of PDS Registry and its main components is shown below.
-
-.. image:: _static/images/registry-arc.png 
+The PDS Registry is both the Service which provides the functions listed above overall the different nodes of the PDS in an integrated way, and it is the software Application which implements these functions at the node level.
 
+This documentation is intended for the users and adminitrators who contributes to the PDS Registry Service. The users of the PDS Registry Service through the web API should refer to the `PDS Search API user's guide <https://nasa-pds.github.io/pds-api/>`_
```
Comment (ramesh-maddegoda) [inline docs/source/index.rst:13]: Minor typo: "This documentation is intended for the users and **adminitrators** who **contributes** to the PDS Registry Service" should be "This documentation is intended for the users and **administrators** who **contribute** to the PDS Registry Service"

File: docs/source/cloud_startup.rst [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+============================================
+
+Overview
+********
+
+This page provides the steps necessary to get going with data ingestion and
+querying of the PDS Registry Service on the cloud from the perspective of a Discipline
+Node (DN).
+
+Initial set-up
```
Comment (ramesh-maddegoda) [inline docs/source/cloud_startup.rst:12]: Minor typo: Better to have **set-up** start with an upper case S (as **Set-up**)to make this heading consistent with the Title Case used in other headings in this page.

File: docs/source/install/docker-compose.rst [hunk 7]
```diff
@@ -0,0 +1,275 @@
...
+********
+
+All the PDS Registry components are containerized as docker images and published in https://hub.docker.com/u/nasapds.
+
+The container based deployment of the PDS Registry components is orchestrated by Docker Compose, a tool for defining and running multi-container Docker applications.
+
+Therefore, the deployment procedures described here can be used to deploy server-side and/or client-side registry components, without having to install the individual components manually.
+
+.. warning::
+ these deployment are not intended for production given that 1. docker compose works on a single host and 2. permanent storage is left to docker to manage.
```
Comment (ramesh-maddegoda) [inline docs/source/install/docker-compose.rst:15]: Minot suggestion: It is better to start this warning with a upper case T (These) to make it consistent with the .. note: below.

File: docs/source/install/docker-compose.rst [hunk 8]
```diff
@@ -0,0 +1,275 @@
...
+
+    b) Generate self-signed certificates by executing the following script. ::
+
+        ./generate-certs.sh
+
+    c) Change the current working directory to `<REGISTRY_ROOT>/docker` directory. ::
+
+        cd ..
+
+Deployment profiles
```
Comment (ramesh-maddegoda) [inline docs/source/install/docker-compose.rst:86]: Suggestion: It is better to start the work profiles with upper case P (Profiles) in this heading to be consistent with the [Title Case ](https://en.wikipedia.org/wiki/Title_case)used on most of the heading in the same heading level in this page (E.g.: Clean-up the Deployment, Download and Configure the PDS Registry Application)

File: docs/source/install/docker-compose.rst [hunk 9]
```diff
@@ -0,0 +1,275 @@
...
+                                                                             (profiles to start before this)
+
+====================== ==================================================== ==============================================
+ pds-core-registry      Starts only the OpenSearch and Registry API          None
+ pds-loader-services    Starts the Scalable Harvest server-side components   The `pds-core-registry` profile
+                                                                             must be up and running
+====================== ==================================================== ==============================================
+
+
+Client-side operations
```
Comment (ramesh-maddegoda) [inline docs/source/install/docker-compose.rst:119]: Title Case Suggestion:  **Client-side Operations**

It is better to use  the  [Title Case ](https://en.wikipedia.org/wiki/Title_case) in this heading to be consistent with the Title Cases used on most of the headings in this page.

File: docs/source/install/docker-compose.rst [hunk 10]
```diff
@@ -0,0 +1,275 @@
...
+ pds-batch-loader.sh  <harvest job confg file>     Executes the Standalone Harvest client-side tool.    The `pds-core-registry` server-side profile
+                                                   This tool is recommended for small data sets of      must be up and running
+                                                   up to 10,000 PDS4 labels.
+ pds-service-loader.sh <harvest job confg file>    Executes the Scalable Harvest client-side tool.      The `pds-service-loader` server-side profile
+                                                   This tool is   recommended for larger data sets of   must be up and running
+                                                   over 10,000 PDS4 labels.
+================================================ ==================================================== ==============================================
+
+
+Common deployment scenarii
```
Comment (ramesh-maddegoda) [inline docs/source/install/docker-compose.rst:140]: [Title Case ](https://en.wikipedia.org/wiki/Title_case) Suggestion:  **Common Deployment Scenarii**

File: docs/source/install/docker-compose.rst [hunk 11]
```diff
@@ -0,0 +1,275 @@
...
+    docker-registry-api-1        |   .   ____          _            __ _ _
+    docker-registry-api-1        |  /\\ / ___'_ __ _ _(_)_ __  __ _ \ \ \ \
+    docker-registry-api-1        | ( ( )\___ | '_ | '_| | '_ \/ _` | \ \ \ \
+    docker-registry-api-1        |  \\/  ___)| |_)| | | | | || (_| |  ) ) ) )
+    docker-registry-api-1        |   '  |____| .__|_| |_|_| |_\__, | / / / /
+    docker-registry-api-1        |  =========|_|==============|___/=/_/_/_/
+    docker-registry-api-1        |  :: Spring Boot ::        (v2.3.1.RELEASE)
+    docker-registry-api-1        |
+
+Run Standalone Harvest to load some PDS4 products
```
Comment (ramesh-maddegoda) [inline docs/source/install/docker-compose.rst:172]: [Title Case ](https://en.wikipedia.org/wiki/Title_case)Suggestion: **Run Standalone Harvest to Load Some PDS4 Products**

File: docs/source/install/docker-compose.rst [hunk 12]
```diff
@@ -0,0 +1,275 @@
...
+-----------------------------------------------------------
+
+1) Open a new terminal and change the current working directory to the ``<REGISTRY_ROOT>/docker/scripts`` directory.
+
+2) Define an harvest job configuration as specified in :doc:`Harvest Job Configuration<../user/harvest_job_configuration>`
```
Comment (ramesh-maddegoda) [inline docs/source/install/docker-compose.rst:177]: Minor typo: "an harvest" should be "a harvest"

File: docs/source/install/docker-compose.rst [hunk 13]
```diff
@@ -0,0 +1,275 @@
...
+
+3) Run the harvest command:
+
+    ./pds-batch-loader.sh  <harvest job confg file>
```
Comment (ramesh-maddegoda) [inline docs/source/install/docker-compose.rst:181]: Better to use a code block here.

File: docs/source/install/docker-compose.rst (hunk 12 above)
Comment (ramesh-maddegoda) [inline docs/source/install/docker-compose.rst:177]: The following  link did not appear in the generated html.

:doc:`Harvest Job Configuration<../user/harvest_job_configuration>`

File: docs/source/install/docker-compose.rst [hunk 14]
```diff
@@ -0,0 +1,275 @@
...
+4) Start the `pds-loader-services` components as follows to start Scalable Harvest components. ::
+
+    docker compose --profile=pds-loader-services up
+
+Run Scalable Harvest Service client to load some PDS4 products
+-----------------------------------------------------------
+
+1) Open a new terminal and change the current working directory to the ``<REGISTRY_ROOT>/docker/scripts/`` directory.
+
+2) Define an harvest job configuration as specified in :doc:`Harvest Job Configuration<../user/harvest_job_configuration>`
```
Comment (ramesh-maddegoda) [inline docs/source/install/docker-compose.rst:243]: The following  link did not appear in the generated html.

:doc:`Harvest Job Configuration<../user/harvest_job_configuration>`

Comment (ramesh-maddegoda) [inline docs/source/install/docker-compose.rst:243]: Minor typo: "an harvest" should be "a harvest"

File: docs/source/install/docker-compose.rst [hunk 15]
```diff
@@ -0,0 +1,275 @@
...
+
+3) Run the harvest command:
+
+    ./pds-service-loader.sh  <harvest job confg file>
```
Comment (ramesh-maddegoda) [inline docs/source/install/docker-compose.rst:247]: Better to use a code block here for,

```
./pds-service-loader.sh  <harvest job confg file>
```

File: docs/source/install/docker-compose.rst [hunk 16]
```diff
@@ -0,0 +1,275 @@
...
+Clean-up the Deployment
+------------------------
+
+The docker containers deployed above can be easily uninstalled and cleaned-up using the following
+commands::
+
+    docker compose --profile=pds-core-registry down
+
+
+Core registry with Scalable Harvest
```
Comment (ramesh-maddegoda) [inline docs/source/install/docker-compose.rst:203]: [Title Case ](https://en.wikipedia.org/wiki/Title_case)Suggestion: **Core Registry with Scalable Harvest**

File: docs/source/install/docker-compose.rst [hunk 17]
```diff
@@ -0,0 +1,275 @@
...
+    docker-registry-api-1        |   '  |____| .__|_| |_|_| |_\__, | / / / /
+    docker-registry-api-1        |  =========|_|==============|___/=/_/_/_/
+    docker-registry-api-1        |  :: Spring Boot ::        (v2.3.1.RELEASE)
+    docker-registry-api-1        |
+
+4) Start the `pds-loader-services` components as follows to start Scalable Harvest components. ::
+
+    docker compose --profile=pds-loader-services up
+
+Run Scalable Harvest Service client to load some PDS4 products
```
Comment (ramesh-maddegoda) [inline docs/source/install/docker-compose.rst:238]: [Title Case ](https://en.wikipedia.org/wiki/Title_case)Suggestion: **Run Scalable Harvest Service Client to Load Some PDS4 Products**

File: docs/source/install/tools.rst [hunk 18]
```diff
@@ -10,10 +10,57 @@ This document describes how to install following PDS Registry tools (command-lin
...
+*************
+
+Java is required to run many Registry components and tools such as Harvest, Registry Manager,
+and API Server.
+
+Only **Java 11** is supported.
+Some applications, like Harvest or Registry Manager will not run with **Java 1.8**.
+
+
+Test if Java is already installed
```
Comment (ramesh-maddegoda) [inline docs/source/install/tools.rst:27]: [Title Case ](https://en.wikipedia.org/wiki/Title_case)Suggestion: **Test If Java Is Already Installed**

This was a tough one. I used the https://titlecaseconverter.com/ get the correct suggestion :-)

File: docs/source/cloud_startup.rst [hunk 19]
```diff
@@ -0,0 +1,90 @@
...
+  * A list of IP addresses or blocks of addresses from which data ingestion will occur.
+
+Opensearch accounts will be set up for each user and their username and (temporary)
+password will be sent to them via JPL Large File Transfer (LFT) which provides secure
+transit of the credentials. The IP addresses are added to the Opensearch access whitelist.
+
+Along with your Opensearch user credentials you will receive the URL which points to your Opensearch
+instance - we will refer to this as the Opensearch URL.
+
+Changing Your Opensearch password
```
Comment (ramesh-maddegoda) [inline docs/source/cloud_startup.rst:27]: [Title Case ](https://en.wikipedia.org/wiki/Title_case)Suggestion: **Changing Your Opensearch Password**

File: docs/source/cloud_startup.rst [hunk 20]
```diff
@@ -0,0 +1,90 @@
...
+  #. In the pop-up window which opens, enter your current password, new password and confirmation of the new password. Click "Reset". (If the Reset button does not activate, it means the password and its confirmation do not match.)
+
+.. image:: /_static/images/kibana_password_reset.png
+
+
+Registry Tools
+**************
+
+Once the above setup is complete, the Registry Tools are used to process and ingest product
+metadata into the Registry. Details on how to run these tools is found in the :doc:`User
```
[the remark this answers is not in the document]
Comment (tloubrieu-jpl) [reply docs/source/cloud_startup.rst:54]: Done

File: docs/source/about.rst (hunk 1 above)
Comment (tloubrieu-jpl) [reply docs/source/about.rst:15]: I missed to add it to git. Done now.

File: docs/source/admin/tasks.rst (hunk 2 above)
Comment (tloubrieu-jpl) [reply docs/source/admin/tasks.rst:5]: Done

[the remark this answers is not in the document]
Comment (tloubrieu-jpl) [reply docs/source/admin/tasks.rst:5]: Done

File: docs/source/cloud/architecture.rst (hunk 3 above)
Comment (tloubrieu-jpl) [reply docs/source/cloud/architecture.rst:8]: I tried to simplify the sentence, these are not my words so I am not necessarily comfortable in doing that but I agree this sentence is a bit complicated, although it sounds like good english.

File: docs/source/cloud/architecture.rst (hunk 4 above)
Comment (tloubrieu-jpl) [reply docs/source/cloud/architecture.rst:42]: What about "Cloudfront provides high-speed, low-latency
access while also preventing from unintentional or malicious bulk download
of data" ?

File: docs/source/index.rst (hunk 5 above)
Comment (tloubrieu-jpl) [reply docs/source/index.rst:13]: Done

File: docs/source/cloud_startup.rst (hunk 6 above)
Comment (tloubrieu-jpl) [reply docs/source/cloud_startup.rst:12]: Done

File: docs/source/install/docker-compose.rst (hunk 7 above)
Comment (tloubrieu-jpl) [reply docs/source/install/docker-compose.rst:15]: Ok, done

File: docs/source/install/docker-compose.rst [hunk 21]
```diff
@@ -0,0 +1,275 @@
...
+================================================ ==================================================== ==============================================
+ Command prototype                                 Description                                          Prerequisites
+
+                                                                                                        (profiles to start before this)
+
+================================================ ==================================================== ==============================================
+ pds-batch-loader.sh  <harvest job confg file>     Executes the Standalone Harvest client-side tool.    The `pds-core-registry` server-side profile
+                                                   This tool is recommended for small data sets of      must be up and running
+                                                   up to 10,000 PDS4 labels.
+ pds-service-loader.sh <harvest job confg file>    Executes the Scalable Harvest client-side tool.      The `pds-service-loader` server-side profile
```
[the remark this answers is not in the document]
Comment (tloubrieu-jpl) [reply docs/source/install/docker-compose.rst:134]: Maybe I introduced that mistake, anyway it is corrected now.

File: docs/source/install/docker-compose.rst (hunk 8 above)
Comment (tloubrieu-jpl) [reply docs/source/install/docker-compose.rst:86]: Done

File: docs/source/install/docker-compose.rst (hunk 9 above)
Comment (tloubrieu-jpl) [reply docs/source/install/docker-compose.rst:119]: Done

File: docs/source/install/docker-compose.rst (hunk 10 above)
Comment (tloubrieu-jpl) [reply docs/source/install/docker-compose.rst:140]: done

File: docs/source/install/docker-compose.rst (hunk 11 above)
Comment (tloubrieu-jpl) [reply docs/source/install/docker-compose.rst:172]: Done

File: docs/source/install/docker-compose.rst (hunk 12 above)
Comment (tloubrieu-jpl) [reply docs/source/install/docker-compose.rst:177]: Done

File: docs/source/install/docker-compose.rst (hunk 13 above)
Comment (tloubrieu-jpl) [reply docs/source/install/docker-compose.rst:181]: Done

File: docs/source/install/docker-compose.rst (hunk 12 above)
Comment (tloubrieu-jpl) [reply docs/source/install/docker-compose.rst:177]: I missed to add the file on git. This done now

File: docs/source/install/docker-compose.rst (hunk 14 above)
Comment (tloubrieu-jpl) [reply docs/source/install/docker-compose.rst:243]: I missed to add the file on git. This done now

Comment (tloubrieu-jpl) [reply docs/source/install/docker-compose.rst:243]: Done

File: docs/source/install/docker-compose.rst (hunk 15 above)
Comment (tloubrieu-jpl) [reply docs/source/install/docker-compose.rst:247]: ok

Comment (tloubrieu-jpl) [reply docs/source/install/docker-compose.rst:247]: I mean done.

File: docs/source/install/docker-compose.rst (hunk 16 above)
Comment (tloubrieu-jpl) [reply docs/source/install/docker-compose.rst:203]: Done

File: docs/source/install/docker-compose.rst (hunk 17 above)
Comment (tloubrieu-jpl) [reply docs/source/install/docker-compose.rst:238]: Done

File: docs/source/install/tools.rst (hunk 18 above)
Comment (tloubrieu-jpl) [reply docs/source/install/tools.rst:27]: I need to learn that, `to` stays 'to' but 'if' becomes 'If' as well as 'Is'. It is a whole field of expertise that I had no idea of  !

Comment (tloubrieu-jpl) [reply docs/source/install/tools.rst:27]: Well, done

File: docs/source/cloud_startup.rst (hunk 19 above)
Comment (tloubrieu-jpl) [reply docs/source/cloud_startup.rst:27]: Done, I first thought you missed it so I corrected and then I saw your comment !

File: docs/source/install/docker-compose.rst (hunk 21 above)
[the remark this answers is not in the document]
Comment (ramesh-maddegoda) [reply docs/source/install/docker-compose.rst]: No, that was my mistake.

File: docs/source/cloud/architecture.rst (hunk 4 above)
Comment (ramesh-maddegoda) [reply docs/source/cloud/architecture.rst:42]: Your suggestion is good!

**Pull request merged**